skimage/transform/_geometric.py

```diff
@@ -6,15 +6,6 @@
 from .._shared.utils import get_bound_method_class, safe_as_int
 
 
-def _to_ndimage_mode(mode):
```

Comment:
I see in master branch that `_to_ndimage_mode` is not used in this file but we are reimporting it from `skimage._shared.utils` :confounded:

Reply:
is this just a comment or are you recommending something here? (I didn't quite follow where we are "reimporting it from")

Reply:
It was just a comment :slightly_smiling_face: